.gitmodules

```diff
@@ -5,7 +5,7 @@
 [submodule "deepfence_agent/plugins/package-scanner"]
 	path = deepfence_agent/plugins/package-scanner
 	url = https://github.com/deepfence/package-scanner
-	branch = main
+	branch = update-tools
```

Comment:
revert this to main when this PR is merged https://github.com/deepfence/package-scanner/pull/38